pkg/kapp/diff/change.go

```diff
@@ -77,6 +82,9 @@ func (d *ChangeImpl) AppliedResource() ctlres.Resource  { return d.appliedRes }
 
 func (d *ChangeImpl) Op() ChangeOp {
```

Comment:
can this method be rearranged to only have one call to hasExternalResourceAnnotation?

Reply:
I will try to do that without compromising readability.

Reply:
Done